src/Avalonia.Themes.Fluent/ComboBox.xaml

```diff
@@ -1,64 +1,191 @@
-            </ToggleButton>
-            <Popup Name="PART_Popup"
-                   IsOpen="{TemplateBinding IsDropDownOpen, Mode=TwoWay}"
-                   MinWidth="{Binding Bounds.Width, RelativeSource={RelativeSource TemplatedParent}}"
-                   MaxHeight="{TemplateBinding MaxDropDownHeight}"
-                   PlacementTarget="{TemplateBinding}"
-                   StaysOpen="False">
-              <Border BorderBrush="{DynamicResource ThemeBorderMidBrush}"
-                      BorderThickness="1">
-                <ScrollViewer>
-                  <ItemsPresenter Name="PART_ItemsPresenter"
-                                  Items="{TemplateBinding Items}"
-                                  ItemsPanel="{TemplateBinding ItemsPanel}"
-                                  ItemTemplate="{TemplateBinding ItemTemplate}"
-                                  VirtualizationMode="{TemplateBinding VirtualizationMode}"
+                          Margin="0,1,1,1"
+                          Width="30" IsVisible="False"
+                          HorizontalAlignment="Right" />
+
+          <Viewbox UseLayoutRounding="False"
+            MinHeight="{DynamicResource ComboBoxMinHeight}"
+              Grid.Row="1"
+              Grid.Column="1"
+              IsHitTestVisible="False"
+              Margin="0,0,10,0" Height="12" Width="12"
+              HorizontalAlignment="Right"
+              VerticalAlignment="Center">
+            <Panel>
+              <Panel Height="12" Width="12" />
+              <Path x:Name="DropDownGlyph" Stretch="Uniform" VerticalAlignment="Center" Data="M1939 486L2029 576L1024 1581L19 576L109 486L1024 1401L1939 486Z" />
+            </Panel>
+          </Viewbox>
+          <Popup Name="PART_Popup"
+                 WindowManagerAddShadowHint="False"
+                 IsOpen="{TemplateBinding IsDropDownOpen, Mode=TwoWay}"
+                 MinWidth="{Binding Bounds.Width, RelativeSource={RelativeSource TemplatedParent}}"
+                 MaxHeight="{TemplateBinding MaxDropDownHeight}"
+                 PlacementTarget="{TemplateBinding}"
+                 StaysOpen="False">
+            <Border x:Name="PopupBorder"
+                    Background="{DynamicResource ComboBoxDropDownBackground}"
+                                BorderBrush="{DynamicResource ComboBoxDropDownBorderBrush}"
+                                BorderThickness="{DynamicResource ComboBoxDropdownBorderThickness}"
+                                Margin="0,-1,0,-1"
+                                Padding="{DynamicResource ComboBoxDropdownBorderPadding}"
+                                HorizontalAlignment="Stretch"
+                                CornerRadius="{DynamicResource OverlayCornerRadius}">
+              <ScrollViewer>
+                <ItemsPresenter Name="PART_ItemsPresenter"
+                                Items="{TemplateBinding Items}"
```

Comment:
I think there is missed Margin. It exists in [WinUI's style](https://github.com/microsoft/microsoft-ui-xaml/blob/master/dev/ComboBox/ComboBox_themeresources.xaml#L695) and has value "0,4,0,4".
```suggestion
                                Items="{TemplateBinding Items}"
                                Margin="{DynamicResource ComboBoxDropdownContentMargin}"
```